src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -34,6 +34,7 @@
 public class ArgsParserTest {
 
     private static final Path PROJECT_DIRECTORY = Paths.get(System.getProperty("user.dir"));
+    private static final Path CONFIG_DIRECTORY = Paths.get(System.getProperty("user.dir") + "/config/");
```

Comment:
would this work in windows machines? perhaps use File.separator instead?

Reply:
Aah sorry. Forgot to do that. Thanks for catching it!